packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -5,12 +5,12 @@ import {
   AmplifyUserError,
 } from '@aws-amplify/platform-core';
 import { BackendDeployerOutputFormatter } from './types.js';
-import { EOL } from 'os';
 
 /**
  * Transforms CDK error messages to human readable ones
  */
 export class CdkErrorMapper {
+  private eolRegex = '[\r\n]+';
```

Comment:
```suggestion
  private multiLineEolRegex = '[\r\n]+';
```

Because it would match the whole space between these ?

```
foo


bar
```

Reply:
Precise regex for single eol would be `\r\n?|\n`

Reply:
that's correct, updated the variable name.